src/lib/api/services/extension.service.ts

```diff
@@ -14,12 +16,15 @@ type UpdateLaunchURLPayload = {
 
 export default {
     registerExtension: async (
-        partner_access_token: string,
-        data: RegisterExtensionPaylaod,
+        data: RegisterExtensionPayload,
     ) => {
         try {
             let headers = getCommonHeaderOptions().headers;
-            headers['x-partner-token'] = partner_access_token;
+            data.scope = ['company/profile'];
+            data.logo = { 
+                small: 'https://res.cloudinary.com/dwzm9bysq/image/upload/v1566539375/production/media/store/logo/jwosxsgh9ufoucdxpm10.png',
+                large: 'https://res.cloudinary.com/dwzm9bysq/image/upload/v1566539375/production/media/store/logo/jwosxsgh9ufoucdxpm10.png'
+            }
```

Comment:
why not passing callbackes too?

Reply:
developed_by_name and contact_email also

Reply:
1. callbacks will be handled at api level
2. we are not taking that info from users when creating extension with cli

Reply:
As it is available on cli as well as  editable in future from developer side we are adding  from cli of whichever account they have logged in will be using that only